qiskit_experiments/framework/composite/composite_analysis.py

```diff
@@ -114,6 +148,10 @@ def _run_analysis(self, experiment_data: ExperimentData):
         # the parent experiment analysis results
         for sub_expdata in component_expdata:
             sub_expdata.block_for_results()
+        # Optionally combine results from all component experiments
+        # for adding to the main experiment data container
+        if self.options.combine_results:
+            return self._combine_results(component_expdata)
```

Comment:
If the parent has results, as in Tphi, will they be part of the flattened results? Can you add a test for this case?